classes/core-image-pelux-qtauto.bbclass

```diff
@@ -0,0 +1,21 @@
+#
+#   Copyright (C) 2017 Pelagicore AB
+#   SPDX-License-Identifier: MIT
+#
+
+DESCRIPTION = "Class for inheriting a Pelux image with Qt frontend"
+
+inherit core-image-pelux
+
+# A bunch of QtAuto components
+QTAUTO_COMPONENTS := " \
```

Comment:
`:=` is not necessary I think.. `=` should work, since there are no variables to expand?

Reply:
Right, `:=` gives immediate expansion, but it is not necessary here. I was poking around with it earlier when it didn't work, but forgot to revert.